docs/handbook/systems-administration.md

```diff
@@ -42,32 +42,44 @@ tasks can be carried out.
-privileges, or to _do_ an administrative command sudo(1m) can be used.
-
-However, security concerns dictate that performing sensitive administration
-tasks would be more secure if carried out by a user with a minimum number of
-privileges. Both aforementioned mechanisms provide _all_ privileges.
-
-It is not always prudent to perform system administration duties with all
+OpenIndiana is a multi-user platform. Performing administrative tasks on a
+multi-user platform can affect all users, indeed if not carried out correctly,
+dire consequences for all users can result. Sensitive administrative commands
+were originally restricted to a single user account, and only a select number of privileged
+users were provided access to this account.  This special user account is known
+as the _superuser_ or _root_ user from which susceptible administrative
+operations could be executed. The superuser account is assigned _all_
 privileges.
 
-Hence, a mechanism was developed whereby users could be assigned a select number
-of privileges by the superuser. OpenIndia provides, in addition to these
-traditional  mechanisms, a richer means to perform these duties known as
-Role-Based Access Control.
+To become root, it is possible to log in directly to the root account or it can
+be accessed from a standard user account by using the command: [su(1M)](https://illumos.org/man/1M/su).
+
+Assigning one or more users all privileges has its limitations, most critically
+concerning security. Tightening security dictate that performing sensitive
+administration tasks would be more secure if carried out by a user with a
+minimum number of privileges.  Ideally, a user should only have sufficient
+privileges required to perform a task and no more. Moreover, it is not always
+necessary for a single user to perform all administrative tasks. It would be
+more flexible if some tasks could be performed by some, say, experienced users.
+
+One scheme adopted to improve the situation was to enable some users
+to carry one or a select number of commands normally requiring elevated
+privileges, or to _do_ an administrative command sudo(1m) can be used. In other
+words, sensitive commands can be assigned to a user and the user is allowed to
+execute those select sensitive commands and no more.
+
+An alternative mechanism was developed focused on the idea of _roles_ and
+_privileges_. A role can be thought of a collection of related tasks. Some roles
+might be printer administration, network responsibility, software installation,
+etc. Associated with such roles are a set of _privileges_ necessary to perform
+the duties associated with these roles. This is known as _Role-Based Access
+Control_ (RBAC). It is sometimes also known as Role Based Security.
 
 RBAC involves collecting a select number of privileges and bundling these
 together as a role. A user can then be assigned one, or several roles.
 
+OpenIndia supports all these frameworks.
```

Comment:
perhaps, rephrase, like "OpenIndiana supports both mechanisms for granting subset of privileges to a user".

Reply:
RBAC is work-in-progress and the material already presented will be drastically changed, and a lot more material is required, but for the present text, your suggestion is better. ok